app/models/role.rb

```diff
@@ -15,6 +15,14 @@ def self.student_role
     Role.find_by('role_name like ?', 'Student')
   end
 
+  def self.tutor_role
+    Role.find_by('role_name like ?', 'Tutor')
+  end
+
+  def self.spartan_tutor_role
+    Role.find_by('role_name like ?', 'Spartan Tutor')
+  end
+
```

Comment:
The ReadMe makes it sound like these are substitutes for "tutor?" and "spartan_tutor?", etc. However, this doesn't seem to fulfill the same role. While these methods do seem useful, why were they used in place of the checker functions? Those methods provided a function that we very frequently need (checking if a user has a certain role) and does so quicker, easier, and with more readability than using these functions would appear to, unless I'm missing something.

Reply:
These methods were created by Nick at the same time I created 'admin?' etc. There was a good reason I switched over to using Nick's version, but that was several weeks ago.

Reply:
What I mean is that these don't seem to fulfill the same role as the methods they "replace". For instance, `tutor?` checks if a given user is a tutor and returns the corresponding boolean. `tutor_role` returns the Role record going by the tutor name. Unless I'm misunderstanding something, they don't fulfill the same function, so why did we deprecate `tutor?` and its related methods?

Reply:
I think I had a misunderstanding of how Nick's version worked until I went to write tests for them. they work well if you're looking for a true value, less so if you're looking for false.

Reply:
I'm not entirely sure how these are even "versions", as they fulfill _completely_ different functions... but I suppose I'll just trust they work for now and add compatibility when I do my PR.

Reply:
if I don't run into any major issues, I'll add them back in